products/compute/api.yaml

```diff
@@ -5841,6 +5841,170 @@ objects:
         description: |
           URL of the disk type resource describing which disk type to use to
           create the disk. Provide this when creating the disk.
+  - !ruby/object:Api::Resource
+    name: 'ResourcePolicy'
+    min_version: beta
+    kind: 'compute#resourcePolicy'
+    base_url: projects/{{project}}/regions/{{region}}/resourcePolicies
+    input: true
+    has_self_link: true
+    description: |
+      A policy that can be attached to a resource to specify or schedule actions on that resource.
+    async: !ruby/object:Api::Async
+      operation: !ruby/object:Api::Async::Operation
+        kind: 'compute#operation'
+        path: 'name'
+        base_url: 'projects/{{project}}/global/operations/{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::Async::Result
+        path: 'targetLink'
+      status: !ruby/object:Api::Async::Status
+        path: 'status'
+        complete: 'DONE'
+        allowed:
+          - 'PENDING'
+          - 'RUNNING'
+          - 'DONE'
+      error: !ruby/object:Api::Async::Error
+        path: 'error/errors'
+        message: 'message'
+    parameters:
+      - !ruby/object:Api::Type::ResourceRef
+        name: region
+        resource: Region
+        imports: name
+        description: Region where resource policy resides.
+        input: true
```

Comment:
consistency wise, input would already be default true if the resource is input only, but there's no harm in leaving it